engine/src/main/java/org/terasology/rendering/nui/layouts/CardLayout.java

```diff
@@ -26,20 +26,43 @@
 import java.util.Iterator;
 import java.util.Map;
 
+/**
+ * A layout that allows for a single widget to be displayed among a list of stored widgets
+ */
 public class CardLayout extends CoreLayout<LayoutHint> {
+    /**
+     * The id of the currently displayed widget
+     */
```

Comment:
@emanuele3d: 1. the other class/field/method comments; 2. nope (try replacing any variable's Javadoc with `// test` then Ctrl+Q'ing it) - but a one-line "multiline" comment i.e. `/** test */` will work fine 🙂